action.yml

```diff
@@ -10,7 +10,10 @@ inputs:
     required: true
   commit-hash:
     description: 'The commit hash'
-    required: true
+    required: false
+  diff-id:
+    description: 'Alternative to commit-hash as a unique identifier for visual tests.'
+    required: false
```

Comment:
I did not update readme because I want to test the functionality before updating doc.